### PR TITLE
Kaio Targets

### PR DESCRIPTION
**Pull request by Kaiowarez**
added HGLRCF405 target

### CODE REVIEW

**Comment by Kaiowarez** [comment]
Yes all targets confirmed working

**Comment by nerdCopter** [comment]
All these targets as one PR?

**Comment by gretel** [comment]
@nerdCopter you prefer 11 pull requests?

**Comment by Kaiowarez** [comment]
Yeah all these targets are good and can be added

**Comment by Kaiowarez** [comment]
Got another dozen once confirmed working

**Comment by nerdCopter** [comment]
i'm trying to follow the one feature/bugfix workflow that was presented.
in the end, _i'm good either way_. 
i just thought it best for break-fix possibilities.

**Comment by Kaiowarez** [comment]
Ah figured throw all the known good targets as 1
Then as I work on new targets follow the 1 good 1 pull system we talked
about

**Comment by gretel** [comment]
this is one and the same feature - adding new targets. code quality and integration builds wont validate flight performance in any case.

File: src/main/target/CRASHTESTF7/config.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
```
Comment (gretel) [inline src/main/target/CRASHTESTF7/config.c]: commenting out definitions gets pretty ambigious without commenting why these got disabled. please clean up or add comments

**Comment by Kaiowarez** [comment]
No relations to flight performance at all, just makes emu more accessible
to more pilots

**Comment by gretel** [comment]
@Kaiowarez no worries was talking to @nerdCopter :)

**Comment by Kaiowarez** [comment]
Ahh that was an experiment
For pinio, apparently 40,255,255,255 = just 40

File: src/main/target/CRASHTESTF7/config.c (hunk 1 above)
Comment (Quick-Flash) [reply src/main/target/CRASHTESTF7/config.c]: Yes Kiao if you could make comments why this is commented out or just remove it then I'll be down to get this all merged into the code. Thanks for your work buddy!

**Comment by Kaiowarez** [comment]
I'll fix it now

**Comment by Kaiowarez** [comment]
Ty

**Comment by Kaiowarez** [comment]
Thanks

**Pull request merged**